iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -66,56 +66,73 @@ constexpr units::Duration DISCOVERY_INTERVAL = 100_ms;
-constexpr uint32_t MAX_CHUNKS_ALLOCATE_PER_SENDER = build::IOX_MAX_CHUNKS_ALLOCATE_PER_SENDER;
-constexpr uint64_t MAX_HISTORY_CAPACITY_OF_CHUNK_DISTRIBUTOR = build::IOX_MAX_HISTORY_CAPACITY_OF_CHUNK_DISTRIBUTOR;
-constexpr uint32_t MAX_CHUNKS_HELD_PER_RECEIVER = build::IOX_MAX_CHUNKS_HELD_PER_RECEIVER;
-constexpr uint32_t MAX_RECEIVER_QUEUE_CAPACITY = MAX_CHUNKS_HELD_PER_RECEIVER;
-/// With MAX_RECEIVER_QUEUE_CAPACITY = MAX_CHUNKS_HELD_PER_RECEIVER we couple the maximum number of chunks a user is
+constexpr uint32_t MAX_CHUNKS_ALLOCATED_PER_PUBLISHER_SIMULTANEOUSLY = build::IOX_MAX_CHUNKS_ALLOCATED_PER_PUBLISHER_SIMULTANEOUSLY;
+constexpr uint64_t MAX_PUBLISHER_HISTORY = build::IOX_MAX_PUBLISHER_HISTORY;
+// Subscriber
+constexpr uint32_t MAX_SUBSCRIBERS = build::IOX_MAX_SUBSCRIBERS;
+constexpr uint32_t MAX_CHUNKS_HELD_PER_SUBSCRIBER_SIMULTANEOUSLY = build::IOX_MAX_CHUNKS_HELD_PER_SUBSCRIBER_SIMULTANEOUSLY;
+constexpr uint32_t MAX_SUBSCRIBER_QUEUE_CAPACITY = MAX_CHUNKS_HELD_PER_SUBSCRIBER_SIMULTANEOUSLY;
+/// With MAX_SUBSCRIBER_QUEUE_CAPACITY = MAX_CHUNKS_HELD_PER_SUBSCRIBER_SIMULTANEOUSLY we couple the maximum number of chunks a user is
 /// allowed to hold with the maximum queue capacity.
 /// This allows that a polling user can replace all the held chunks in one execution with all new ones
 /// from a completely filled queue. Or the other way round, when we have a contract with the user
 /// regarding how many chunks they are allowed to hold, then the queue size needs not be bigger. We
 /// can provide this number of newest chunks, more the user would not be allowed to hold anyway
-constexpr uint32_t MAX_INTERFACE_CAPRO_FIFO_SIZE = MAX_PORT_NUMBER;
-constexpr uint32_t MAX_APPLICATION_CAPRO_FIFO_SIZE = 128u;
+// Gateway
+constexpr uint32_t MAX_INTERFACE_NUMBER = build::IOX_MAX_INTERFACE_NUMBER;
+constexpr uint32_t MAX_INTERFACE_CAPRO_FIFO_SIZE = MAX_PUBLISHERS;
+// Client
+constexpr uint32_t MAX_CLIENTS = build::IOX_MAX_SUBSCRIBERS; /// @todo
+constexpr uint32_t MAX_REQUESTS_ALLOCATED_SIMULTANEOUSLY = 4U;
+constexpr uint32_t MAX_RESPONSES_PROCESSED_SIMULTANEOUSLY = 16U;
+constexpr uint32_t MAX_RESPONSE_QUEUE_CAPACITY = 16U;
+// Server
+constexpr uint32_t MAX_SERVERS = build::IOX_MAX_PUBLISHERS; /// @todo
+constexpr uint32_t MAX_CLIENTS_PER_SERVER = 256U;
+constexpr uint32_t MAX_REQUESTS_PROCESSED_SIMULTANEOUSLY = 4U;
+constexpr uint32_t MAX_RESPONSES_ALLOCATED_SIMULTANEOUSLY = MAX_REQUESTS_PROCESSED_SIMULTANEOUSLY;
+constexpr uint32_t MAX_REQUEST_QUEUE_CAPACITY = 1024;
+// Waitset
+constexpr uint32_t MAX_NUMBER_OF_CONDITION_VARIABLES = 1024U;
+constexpr uint32_t MAX_NUMBER_OF_CONDITIONS_PER_WAITSET = 128U;
+//--------- Communication Resources End---------------------
+
+constexpr uint32_t MAX_APPLICATION_CAPRO_FIFO_SIZE = 128U;
 
 // Memory
-constexpr uint64_t SHARED_MEMORY_ALIGNMENT = 32u;
-constexpr uint32_t MAX_NUMBER_OF_MEMPOOLS = 32u;
-constexpr uint32_t MAX_SHM_SEGMENTS = 100u;
+constexpr uint64_t SHARED_MEMORY_ALIGNMENT = 32U;
```

Comment:
I like capital literals. Makes it harder to confuse a "one" and a lower case L. So, that's the new style for iceoryx! 👍